backend/src/rest-api-docs.yaml

```diff
@@ -100,6 +102,23 @@ paths:
       responses:
         200:
           description: Player successfully registered for the tournament
+  /players-profiles:
+    post:
+      tags:
+        - Players profiles
+      description: Create new player profile
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PostPlayerProfileRequestBody'
+      responses:
+        200:
+          description: Player profile successfully created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PostPlayerProfileResponseBody'
```

Comment:
zrobiłeś zapytanie POST, ale już nie GET :P możesz w następnym PR tylko trzeba o tym pamiętać ;)

Reply:
do GET pójdzie osobny PR, bo chyba i tak jeszcze do niego nie mam wszystkiego w application i domain :-) (ale to muszę się zastanowić...tak czy inaczej wiem wiem, że GET nie ma ... jest w kolejce :-) )